libs/components/tabs/testing/src/tabs/tabset-fixture.spec.ts

```diff
@@ -164,11 +164,16 @@ describe('Tabset fixture', () => {
         .nativeElement
     ).toHaveCssClass('sky-btn-tab-selected');
 
-    let errorMessage: string;
+    let errorMessage: string | undefined;
 
     try {
       await tabset.clickTab(100);
     } catch (err) {
+      if (!(err instanceof Error)) {
```

Comment:
We have had to do this in other places recently. `err` is not typed as `Error` and so we have to add this check in order to check `err.message` below.